pkg/k8s/node.go

```diff
@@ -64,6 +66,14 @@ func ParseNode(k8sNode *slim_corev1.Node, source source.Source) *nodeTypes.Node
 			}).Warn("Ignoring invalid node IP")
 			continue
 		}
+		for addrElement = range addr {
+			encountered = addressTypes[addrElement.Type]
+			if encountered {
+				scopedLog.WithError(err).Warn("following address type is repeated for a particular node")
```

Comment:
What err is this referring too?

Also can we log the address/address-type of the offending address.